docs/getting-started/publishing-keys.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+---
+
+# Key Publishing Guide
+
+This page explains how to configure the server and application to allow
+publishing keys.
+
+## Prerequisites
+
+If you haven't started developing a mobile app, please take a look at the
+[Android Exposure Notifications Reference App](https://github.com/google/exposure-notifications-android)
+
+You will need to
+[complete the necessary steps](https://support.google.com/googleplay/android-developer/contact/expo_notif_api)
+to use the exposure notifcations API. You should also ensure that you have
+taken the steps to comply with the
+[Terms of Service](https://blog.google/documents/72/Exposure_Notifications_Service_Additional_Terms.pdf).
+For instance, your app may not request any of the following permissions:
+  * ACCESS_FINE_LOCATION
+  * ACCESS_COARSE_LOCATION
+  * ACCESS_BACKGROUND_LOCATION
+  * BLUETOOTH_ADMIN
+
+
+## Publishing Keys
+
+In order for your application to publish keys to the server, the server
+requires the addition of the Application Name (for Android) and the Bundle ID
+(for iOS). This can be done via the admin interface.
+
+While not required, if a verification server is required, additional
```

Comment:
this is only optional in a theoretical sense. If you don't do this, the publish endpoint is wide open unless some other type of access control is applied.

Reply:
Yep. may be a good idea to make that clear in this. Will be elaborating and adding some details on admin interface here as well.